test/server/listener_manager_impl_test.h

```diff
@@ -37,7 +37,7 @@ class ListenerHandle {
 
 class ListenerManagerImplTest : public testing::Test {
 protected:
-  ListenerManagerImplTest() : api_(Api::createApiForTest()) {
+  ListenerManagerImplTest() : api_(Api::createApiForTest()), real_listener_factory_(server_) {
```

Comment:
Can you just define this in the test where you use it for now as we don't use it anywhere else?